casts/countryPicker/package.json

```diff
@@ -66,7 +66,8 @@
         "react-native-gesture-handler": "^1.10.3",
         "react-native-reanimated": "^2.2.1",
         "react-native-redash": "^16.0.11",
-        "react-native-svg": "^12.1.1"
+        "react-native-svg": "^12.1.1",
+        "react-native-android-keyboard-adjust": "^1.2.0"
```

Comment:
This should also be added to the `devDependencies` (otherwise TS / ESLint should actually complain...)